README.md

```diff
@@ -1,2 +1,9 @@
-# spring-blog
+# Spring Boot From Zero To Hero
```

Comment:
This needs to be changed to "Spring Blog" and also can remove the link to YouTube video.

Reply:
solved